### DISCUSSION TITLE
Mermaid not working in wikis

### DISCUSSION THREAD

**Discussion by shaunc** [community/community#11820] [Other Feature Feedback, Questions, & Ideas] [answered]
Thank you for mermaid support!  https://github.com/github/roadmap/issues/372

However, mermaid diagrams are not working in wikis -- e.g. https://github.com/FactFiber/kedro-dvc/wiki

As a side note, there seems to be no place to open issues about broken github features. Is this by intent? If not, I would suggest better documentation explaining how, as standard "issues" brings me here to feedback.

**Comment by cveld** [comment]
How is Google Search doing this. I was looking for this broken Mermaid support on GitHub wikis and I found your discussion item :-)

The page:
https://github.com/cveld/markdown-tester/wiki/Mermaid-tester-2

Just contains the following:
```mermaid
  graph TD;
      A-->B;
      A-->C;
      B-->D;
      C-->D;
```

Interestingly it gets rendered perfectly on this discussion thread. But on the wiki page it doesn't.

**Reply by shaunc** [reply]
Yup! ... hopefully they just forgot to throw the switch! :)

**Reply by cveld** [reply]
Would be cool to see so much more being enabled on markdown and wiki! With a theoretical extension model any vendor could plug their microformats. E.g. the actual status of Service Now tickets, or Azure Boards tickets, or metadata from SharePoint files, etc.

**Reply by ferrao** [reply]
I was soooooo excited when I saw [this](https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/), only to find out it does not work where I need it the most... In the Wiki.... Any chance we can get it soon @ankneis ?

**Reply by michellemerrill** [reply]
Hi all 👋🏼 There's a [wiki /mermaid feedback thread](https://github.com/github/feedback/discussions/11612) already started, locking this discussion to prevent duplication.

**Comment by davidstaheli** [comment, marked as answer]
Mermaid (and other fun things) are now supported in wikis! 🎉 See the [changelog](https://github.blog/changelog/2022-08-09-wikis-now-support-math-and-mermaid-diagrams/) for more info.

Enjoy! 🙇
